packages/core/src/postProcess/PostProcessEffectParameter.ts

```
@@ -0,0 +1,94 @@
+  }
+
+  set value(value: T) {
+    if (this.value?.constructor === Number) {
+      this._value = <T>(<unknown>MathUtil.clamp(<number>value, this._min, this._max));
+    } else {
+      this._value = value;
+    }
+  }
+
+  constructor(value: Exclude<T, number>, needLerp?: boolean);
+  constructor(value: Exclude<T, Boolean | Color | Vector2 | Vector3 | Vector4 | Texture>, needLerp?: boolean);
```

Biome (1.9.4)

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)